src/outputProcessor/consoleProcessor.ts

```diff
@@ -127,8 +128,8 @@ function consoleProcessor(
     severity,
     timeString
   }) => {
-    let processedMessage = message;
-
+    let {isItalic, isBold, color: messageColor, processedMessage} = utils.checkMessageMarkdown(message);
```

Comment:
I think this can be expensive with all the regex matching. Can we do this checks more efficiently? Like only when log type is cy.log, for example?

Reply:
Good call!